src/Packetery/Module/Framework/WcSessionTrait.php

```diff
@@ -4,14 +4,34 @@
 
 namespace Packetery\Module\Framework;
 
+use WC_Session;
+use WC_Session_Handler;
+
 trait WcSessionTrait {
 	/**
-	 * @param string $key
-	 * @param mixed  $value
-	 *
-	 * @return void
+	 * @return WC_Session|WC_Session_Handler|null
 	 */
+	public function session() {
+		return WC()->session;
+	}
+
+	/**
+	 * @return array|string
+	 */
+	public function sessionGet( string $key ) {
+		return WC()->session->get( $key );
```

Comment:
nejsem si jistý jestli to nemůže v určitých případech i vrátit false

Reply:
a vlastně i null

Reply:
kopíruji jejich komentář a zatím jsem nenarazil na problém

Reply:
To je právě to, že moc nemůžeme spoléhat na to, že to někde ukládáme jako string, že to tak vždycky bude a vždycky to tam bude uloženo.